CHSHGame/Tests.qs

```diff
@@ -117,24 +121,28 @@ namespace Quantum.Kata.CHSHGame {
     // ------------------------------------------------------
     @Test("QuantumSimulator")
     operation T24_BobQuantum_Test () : Unit {
-        using (q = Qubit()) {
-            RotateBobQubit_Reference(false, q);
-            EqualityFactB(BobQuantum(false, q), false, "π/8 from |0⟩ not measured as false");
-            Reset(q);
-
-            X(q);
-            RotateBobQubit_Reference(false, q);
-            EqualityFactB(BobQuantum(false, q), true, "π/8 from |1⟩ not measured as true");
-            Reset(q);
-
-            RotateBobQubit_Reference(true, q);
-            EqualityFactB(BobQuantum(true, q), false, "-π/8 from |0⟩ not measured as false");
-            Reset(q);
-
-            X(q);
-            RotateBobQubit_Reference(true, q);
-            EqualityFactB(BobQuantum(true, q), true, "-π/8 from |1⟩ not measured as true");
-            Reset(q);
+        for _ in (1 .. 4) {
```

Comment:
```suggestion
        for (_ in 1 .. 4) {
```

Reply:
hmm if you go to line 139 I'm following that

Reply:
oops addressed